src/request.js

```diff
@@ -136,4 +136,19 @@ module.exports = class Request extends EventEmitter {
     }
     return null;
   }
+
+  // Temporarily suspends the flow of data from the database.
+  // No more 'row' events will be emitted until resume() is called.
+  pause() {
```

Comment:
Sounds reasonable. Opened https://github.com/tediousjs/tedious/issues/534 to add 'cancel' on Request and deprecate it on Connection.